### ISSUE TITLE
When keyboard shown the message input moves too high and obscures the message feed.

### ISSUE THREAD

**Issue by matthewfx** [GetStream/stream-chat-flutter#740]
### Which packages are you using?

stream_chat_flutter

### On what platforms did you experience the issue?

iOS

### What version are you using?

stream_chat_flutter: ^3.1.1

### What happened?

Hi,

On a real iOS15 iPhone Xs Max (for some reason when testing can't trigger the keyboard on the simulator) when I start typing a message the input field goes to almost the top of the screen and you can't see the message field. Even when you send a message it stays there and the only way of dismissing the keyboard is to swipe down on the input field which is a very bad UX and totally not intuitive. I tested it also on an iPhone 5s with iOS 12.5.5. It is even worse because once I start typing the input field disappears and the news feed looks empty and there is no way to send a message or dismiss a keyboard at all :(
![IMG_8707](https://user-images.githubusercontent.com/3390107/137814299-46224663-90d9-47d8-a7a4-45519e5879db.PNG)
 

### Steps to reproduce

```bash
I have used the code from the tutorial.

https://getstream.io/chat/flutter/tutorial/#add-stream-chat-to-your-flutter-application
```

### Supporting info to reproduce

MaterialApp(
            debugShowCheckedModeBanner: false,
            builder: (context, widget) {
              return StreamChat(
                client: chatData.client!,
                child: widget,
              );
            },
            // home: ChannelListPage()));
            home: StreamChannel(
                channel: chatData.channel!, child: const ChannelPage())));

class ChannelPage extends StatelessWidget {
  const ChannelPage({
    Key? key,
  }) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: const ChannelHeader(),
      body: Column(
        children: const [
          Expanded(
            child: MessageListView(),
          ),
          MessageInput(),
        ],
      ),
    );
  }
}

### Flutter analyze output

```shell
Analyzing ios...
No issues found! (ran in 1.1s)
```

### Flutter doctor output

```shell
[✓] Flutter (Channel stable, 2.5.2, on macOS 11.6 20G165 darwin-arm, locale
    en-AU)
    • Flutter version 2.5.2 at /Users/mat/Flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 3595343e20 (3 weeks ago), 2021-09-30 12:58:18 -0700
    • Engine revision 6ac856380f
    • Dart version 2.14.3

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/mat/Library/Android/sdk
    • Platform android-30, build-tools 30.0.3
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode 12.5.app/Contents/Developer
    • Xcode 12.5.1, Build version 12E507
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6915495)

[✓] VS Code (version 1.61.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.27.0

[✓] Connected device (4 available)
    • iPhone (mobile)            • eac608144aac804c668f2516daa7d52fc5385b30 •
      ios            • iOS 12.5.5 16H62
    • iPhone 12 mini (mobile)    • A28FA5C7-BBC1-4905-A491-E72386745E54     •
      ios            • com.apple.CoreSimulator.SimRuntime.iOS-14-5 (simulator)
    • iPhone 12 Pro Max (mobile) • C4B46F1F-A80A-4500-98E7-20E6A1D72CF2     •
      ios            • com.apple.CoreSimulator.SimRuntime.iOS-14-5 (simulator)
    • Chrome (web)               • chrome                                   •
      web-javascript • Google Chrome 94.0.4606.81
    ! Error: Mat's iPhone is not connected. Xcode will continue when Mat's
      iPhone is connected. (code -13)
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by matthewfx** [comment]
Just found what was causing the above problem. I used your library in a private package. When that module was used within an app the MaterialApp of the Chat component was nested in another MaterialApp. That was apparently resulting in the behaviour described above. I removed the MaterialApp from your tutorial code and it behaves correctly. I'm guessing the only remaining problem for me to solve would be providing a navigator.
